src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -1328,6 +1328,9 @@ public TextLine SplitAt(LineBreak lineBreak, bool keepAll)
 
                 if (index == 0)
                 {
+                    // Now trim trailing whitespace from this line in the case of an exact
+                    // length line break (non CJK)
+                    this.TrimTrailingWhitespaceAndRecalculateMetrics();
```

Comment:
I found that in case of exact length line breaks, there could be whitespace that remains untrimmed - the addition of this line here would take care of that (without losing the benefits of the fix added for issue 367)